tools/test_containers_compose

```diff
@@ -2,11 +2,44 @@
 
 set -euo pipefail
 
-for i in webui worker; do
+wait_until(){
+  command=$1
+  timeout=$2
+
+  i=1
+  until eval "$command"
+  do
+      ((i++))
+      if [[ $i -gt $timeout ]]; then
+          echo "Command '$command' aborted because of timeout"
+          exit 1
+      fi
+      sleep 1
+  done
+}
+
+setup_containers(){
   (
-    cd container/$i/ &&
-     for retry in {2..0}; do sudo docker-compose build && break; echo "Remaining retries $retry"; done &&
-    sudo docker-compose up -d &&
+    for retry in {2..0}; do sudo docker-compose build && break; echo "Remaining retries $retry"; done &&
+    sudo MOJO_CLIENT_DEBUG=1 docker-compose up -d &&
     (docker-compose ps --services --filter status=stopped | grep "^[[:space:]]*$") || (docker-compose logs; sudo docker-compose ps; exit 1)
   ) || exit 1;
-done
+}
+
+#test webui
+(
+  cd container/webui/ &&
+  sed -e "s/method = OpenID/method = Fake/" < conf/openqa.ini > /tmp/openqa.ini &&
```

Comment:
this means that executing the test would change the files in the git checkout, right? I consider this bad style and suggest instead what we follow in other tests: Create a temporary directory and generate files with the correct setup in there, e.g.

```suggestion
conf=$(mktemp -d)
trap 'rm -rf $conf' EXIT
…
  sed -e "s/method = OpenID/method = Fake/" < conf/openqa.ini > $conf/openqa.ini
  …
  export OPENQA_CONFIG=$conf
```

See for example https://github.com/os-autoinst/openQA/blob/master/t/03-auth.t#L32

Reply:
I will copy all files from container/webui to the temp dir, because, also, the docker-compose creates a new directory "workdir"